### ISSUE TITLE
Add support for DMARC verification in SES Events

### ISSUE THREAD

**Issue by fischaz** [aws/aws-lambda-go#63]
as per SES event received under the 'Receipt' struct:
```json
      "receipt": {
        "timestamp": "2018-03-31T01:06:56.352Z",
        "processingTimeMillis": 716,
        "recipients": ["xxxxx@xxxx.com"],
        "spamVerdict": {
          "status": "PASS"
        },
        "virusVerdict": {
          "status": "PASS"
        },
        "spfVerdict": {
          "status": "PASS"
        },
        "dkimVerdict": {
          "status": "PASS"
        },
        "dmarcVerdict": {
          "status": "PASS"
        },
        "action": {
          "type": "Lambda",
          "functionArn": "xxxxxxxxx",
          "invocationType": "RequestResponse"
        }
      }
```

and add the DMARC Verdict in the ses.go event?

```go
type SimpleEmailReceipt struct {
	Recipients           []string                 `json:"recipients"`
	Timestamp            time.Time                `json:"timestamp"`
	SpamVerdict          SimpleEmailVerdict       `json:"spamVerdict"`
	DMARCVerdict         SimpleEmailVerdict       `json:"dmarcVerdict"`
	DKIMVerdict          SimpleEmailVerdict       `json:"dkimVerdict"`
	SPFVerdict           SimpleEmailVerdict       `json:"spfVerdict"`
	VirusVerdict         SimpleEmailVerdict       `json:"virusVerdict"`
	Action               SimpleEmailReceiptAction `json:"action"`
	ProcessingTimeMillis int64                    `json:"processingTimeMillis"`
}
```

**Comment by fischaz** [comment]
Although sadly, the unit test will break with the "current" event that does not contain that entry.

**Comment by bmoffatt** [comment]
addressed by #100
